experiments/tf_trainer/keras_gru_attention/run.py

```diff
@@ -45,17 +51,21 @@ def main(argv):
 
   embeddings_path = FLAGS.embeddings_path
   text_feature_name = FLAGS.text_feature_name
+  key_name = FLAGS.key_name
 
   preprocessor = text_preprocessor.TextPreprocessor(embeddings_path)
-  nltk.download("punkt")
-  tokenize_op = preprocessor.tokenize_tensor_op(nltk.word_tokenize)
+  if FLAGS.preprocess_in_tf:
+    tokenize_op_init = lambda: preprocessor.tokenize_tensor_op_tf_func()
```

Comment:
Can definitely be later work, but we could maybe unify the tf_func and py_func version of this by passing in a tf.py_func in the case we want to use a python function.